src/javaharness/java/arcs/api/Thing.java

```diff
@@ -1,5 +1,7 @@
 package arcs.api;
 
+import java.util.Optional;
+
 public class Thing<T> {
     public final T thing;
 
```

Comment:
add an assertion in constructor that this is never null

Reply:
done